add arch linux install (#119)
pop-os/cosmic-epoch: README.md

# COSMIC Desktop

> Currently an incomplete **pre-alpha**. Standard Installation is available for Pop!_OS, Arch Linux and its decendents, like Manjaro.

Reviewer: I don't mind having specific instructions for some distributions, but in general the systemd-sysext method works on a lot of distributions for testing. Which is all this should be used for right now. So please drop or adjust this line.

```suggestion
Currently an incomplete **pre-alpha**. Testing instructions below for various distributions.
```

* [libcosmic](https://github.com/pop-os/libcosmic)
* [xdg-desktop-portal-cosmic](https://github.com/pop-os/xdg-desktop-portal-cosmic)

> ## Setup without package

Reviewer: ```suggestion
## Setup on distributions without packaging of cosmic components
```

After logging out, click on your user and there will be a sprocket at the bottom right. Change the setting to COSMIC. Proceed to log in.

## Installing on Arch Linux
Installing via the preferred AUR helper is possible the usual way, e.g.:
`paru -S cosmic-epoch-git`

Then log out, click on your user, and a sprocket at the bottom right shows an additional entry alongside your desktop environments. Change to COSMIC and proceed with log in.

> For a more detailled discussion consider the [relevant section in the Arch wiki](https://wiki.archlinux.org/title/COSMIChttps://wiki.archlinux.org/title/COSMIC).

Reviewer: This seems like a copy&paste error?

```suggestion
For a more detailled discussion consider the [relevant section in the Arch wiki](https://wiki.archlinux.org/title/COSMIC).
```
